shared/test_artifacts/gov.go

```diff
@@ -1,9 +1,11 @@
 package test_artifacts
```

Comment:
Non-Blocker:

Can we move this to the runtime package?

Reply:
Done